Polish/przygoda/jedzenie/wiedzmin/placki/igni/igni.txt

```diff
@@ -0,0 +1,2 @@
+Rzucasz zaklecie IGNI - z twoich rak strzelaja plomienie ognia, ktore kierujesz na nerda, przy okazji podpalajac meble stojace wzdluz sciany. Nerd wydaje okropny krzyk. Na jego twarzy pojawiaja sie nagle zmiany. Wydaje ci sie, ze on caly zmienia sie - jego cialo powoli rosnie i zmienia kszta�t. I nagle dociera to do ciebie - on nie jest czlowiekiem. To wampir, w dodatku z gatunku tych wy�szych...
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.